### ISSUE TITLE
Shop address getter compatibility for XML 

### ISSUE THREAD

**Issue by MohamadNateqi** [wpovernight/woocommerce-pdf-invoices-packing-slips#926]
The current getter method for the shop address wraps the address around an HTML P tag. We need to get the content only without any HTML tags for XML usage.

**Comment by MohamadNateqi** [comment]
Removing the P tag from the getter might not be the best approach as it leaves the address inside a DIV:

![image](https://github.com/user-attachments/assets/6c822dbe-c990-4b14-b6b6-26fb0340f562)

Alternative approaches include creating a new getter method for XML or adding a parameter to the existing getter to alter its return format.

**Comment by alexmigf** [comment]
This fix may also need to be applied to other areas, such as customer addresses, company name, and similar fields.

**Comment by MohamadNateqi** [comment]
@alexmigf I think the best approach is to introduce a new parameter to these getters that allows changing the return value format.

Introducing a new getter for each seems unnecessary and excessive, also, using `get_settings_text()` directly might not be the ideal.

Let me know your thoughts!

**Comment by alexmigf** [comment]
We could create a dedicated functions file for UBL-related functionality. It would include a getter for this type of data and could also consolidate other related functions currently in the handler classes. This setup would make things more organized and extension-friendly. The file should load alongside UBL.

**Comment by MohamadNateqi** [comment]
I see that the below method from `OrderDocument` has been used in UBL files:
```php
get_shop_name()
get_shop_address()
get_shop_vat_number()
get_shop_coc_number()
get_number()
get_title()
get_filename()
get_date()
exists()
get_pdf()
```

Among these methods, we only have the issue with the `get_shop_address()` method. The rest of the methods do not return the value wrapped around a P tag.

So, a function file will only include a `get_shop_address` getter. Wouldn't it be better to handle it another way?

**Comment by alexmigf** [comment]
I believe you can include HTML within the strings as well, right? In that case, wouldn't it be better to encapsulate those functions within another function to handle the formatting?

**Comment by alexmigf** [comment]
I planned to start the functions file with a single function and add more later, after merging this. We could follow a similar approach as in this commit: [https://github.com/wpovernight/wpo-ips-xrechnung/commit/71d9fdb8819b6e29ebe1bb0bcf2ee906cd7161cd](https://github.com/wpovernight/wpo-ips-xrechnung/commit/71d9fdb8819b6e29ebe1bb0bcf2ee906cd7161cd).

**Comment by MohamadNateqi** [comment]
> I believe you can include HTML within the strings as well, right?

You're right; in this case, a separate functions file would be better. I'll create it.

Where do you suggest putting this helper functions file? 
I think this path would be good: "ubl -> Handler -> Helpers -> Functions.php"

**Comment by alexmigf** [comment]
> Where do you suggest putting this helper functions file?

How about creating a new file named `wpo-ips-ubl-functions.php` and placing it in the plugin root? It could be loaded alongside the existing `wpo-ips-functions.php` file, ensuring it's available as early as possible for the extensions.

**Comment by MohamadNateqi** [comment]
So a general helper function. Seems good!
I'll do that. Thanks
